cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1511,6 +1521,13 @@ def _add_compute_resource_launch_template(
                                 ).lower(),
                                 "HeadNodePrivateIp": self._head_eni.attr_primary_private_ip_address,
                                 "DirectoryServiceEnabled": str(self._config.directory_service is not None).lower(),
+                                "Timeout": (
+                                    str(get_attr(self._config, "dev_settings.timeouts.compute_node_bootstrap_timeout"))
```

Comment:
simplify this by setting the default in get_attr()

Reply:
Changed